src/index.html

```diff
@@ -9,6 +9,68 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
```

Comment:
change name ti .header. It will be more clear
```suggestion
    <header class="header">
```